engine/sql/183_drop_deprecated_user.sql

```diff
@@ -0,0 +1,120 @@
+-- +migrate Up
+ALTER TABLE "group" ADD COLUMN IF NOT EXISTS sig BYTEA;
+ALTER TABLE "group" ADD COLUMN IF NOT EXISTS signer TEXT;
+
+CREATE TABLE IF NOT EXISTS "group_authentified_user" (
+    id BIGSERIAL PRIMARY KEY,
+    group_id BIGINT NOT NULL,
+    authentified_user_id VARCHAR(36) NOT NULL,
+    group_admin BOOLEAN,
+    sig BYTEA,
+    signer TEXT
+);
+SELECT create_foreign_key_idx_cascade('FK_GROUP_AUTHENTIFIED_USER_USER', 'group_authentified_user', 'group', 'group_id', 'id');
+SELECT create_foreign_key_idx_cascade('FK_GROUP_AUTHENTIFIED_USER_AUTHENTIFIED_USER', 'group_authentified_user', 'authentified_user', 'authentified_user_id', 'id');
```

Comment:
Be careful with cascade, user should not be last admin.

Reply:
it's checked on API Side